hack/verify-govet.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+run_individual_tests() {
+  files_to_test=$1
+  tests="printf methods structtags composites asmdecl assign atomic bool buildtags copylocks nilfunc rangeloops unreachable shadow unsafeptr unusedresult"
+
+  for test in $tests
+  do
+    touch "_output/govet/${test}.log"
+    for file in $files_to_test
+    do
+      go tool vet "-${test}=true" $file 2>&1 | sed '/exit status/d' >> "_output/govet/${test}.log" || true
+    done
+    if [ -s "_output/govet/${test}.log" ]
+    then
+      echo "FAILURE: go vet -${test} had errors: "
```

Comment:
The reason I was throwing things into a logfile and dumping it is that it was nice not to fail on the first instance of a `go vet` failure. It would be nice to have it run once on my PR and show me everything that's wrong -- once we clean things up that doesn't necessarily mean we need to do per-file per-test but it would still be nice not to fail on the first instance.